src/Components/CollectionCard/CollectionCard.js

```diff
@@ -1,18 +1,26 @@
 import React from 'react';
 import imgIcon from '../../assets/icons/image.png';
 import starredIcon from '../../assets/icons/Star.png';
+import collection from '../../Constants/collection.js'
 
-function CollectionCard({ card }) {
-  return (
-  <div className={`bg-${card.type.color}-200 w-80 m-2 p-2 shadow-2xl rounded-xl`}>
+export default function CollectionCard({ card }) {
+  return(
+    collection.map((el) => {
```

Comment:
u can use filter instead

Reply:
is it worth the change? it is working. I changed also the colors in the collection array because tailwind doesn't take hexadecimals in classNames

Reply:
idk if it is worth it :) 
ask Osama

Reply:
it's better to just get the type you have object from the collections array  by using the `filter` or `find` and store in a variable as a const and then use it by saying for example
```jsx
const type = collections.find((collection) => collection.id === card.type.id);
// use the type in the card
```